super_admin_1/errors/handlers.py

```diff
@@ -28,6 +29,19 @@ def custom_error(error):
         error.code,
     )
 
+@error.app_errorhandler(ValidationError)
+def raise_validation_error(error):
+    """app error handler for pydantic validation errors"""
+    msg = []
+    for err in error.errors():
+        msg.append({
+            "field": err["loc"][0],
+            "error": err["msg"]
+        })
```

Comment:
We can just use list comprehension here

```suggestion
    msg = [{"field": err["loc"][0], "error": err["msg"]} for error in errors]

```